flutter_local_notifications/example/lib/main.dart

```diff
@@ -727,6 +727,13 @@ class _HomePageState extends State<HomePage> {
                           await _stopForegroundService();
                         },
                       ),
+                      PaddedElevatedButton(
+                        buttonText:
+                            'Show colored background notification with forgground service',
```

Comment:
For consistency, could you shift this button to be right after the existing "Start foreground service" one? The button text could then become "Start foreground service with notification that has a blue background". This also makes it more intuitive for users to locate the button to stop the foreground service as it would be below the button that starts a foreground service.

`_showColoredBackgroundNotification()` could then be renamed to something like `_startForegroundServiceWithBlueBackgroundNotification()`